fizz_buzz/fizzBuzzQuiz.js

```diff
@@ -0,0 +1,18 @@
+function fizzBuzz(max){
+	for(i=0; i<=max; i++){
+  	if(i % 3 == 0 && i % 5 == 0){
```

Comment:
Also, if you could indent the if a little bit as well it would be easier to read